pydynamo_brain/pydynamo_brain/ui/radiiPainter.py

```diff
@@ -83,63 +84,98 @@ def drawPoint(self, point, selectedPointID):
             )
             self.maybeDrawText(x, y, point)
 
-    def returnRadiusCoord(self, point, radius):
+    def returnRadiusCoord(self, point, radius, realRadius):
```

Comment:
is realRadius used?

It'd be worth having the radius here match the radius used in normal mode when the point doesn't have a radius, too

Reply:
realRadius is used depending on shift-J state